mmv1/templates/terraform/custom_expand/bigquery_table_ref.go.tmpl

```diff
@@ -40,5 +40,19 @@ func expand{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d tpgresource.T
 		transformed["tableId"] =  parts[3]
 	}
 
+	configError := "Invalid BigQuery job destination_table configuration. You must either:\n" +
+	  "1. Set all of project_id, dataset_id, and table_id separately, or\n" +
+		"2. Provide table_id in the form '{{`projects/{{project}}/datasets/{{dataset_id}}/tables/{{table_id}}`}}'"
+
+	// Validate required fields
+	if projectId, ok := transformed["projectId"]; !ok || projectId == nil || 
+	  reflect.ValueOf(projectId).IsZero() {
+		return nil, fmt.Errorf("%s\nMissing or empty projectId", configError)
+	}
+
+	if datasetId, ok := transformed["datasetId"]; !ok || datasetId == nil || 
+	  reflect.ValueOf(datasetId).IsZero() {
+	  return nil, fmt.Errorf("%s\nMissing or empty datasetId", configError)
+	}
 	return transformed, nil
-}
+ }
```

Comment:
Remove unnecessary space?

Reply:
sorry about that. Too used to the auto formatter I suppose!